app/frontend/lib/create-search-model.ts

```diff
@@ -76,6 +76,7 @@ export const createSearchModel = <TSortField, TFetchOptions extends IFetchOption
       applySort(sort: ISort<TSortField>) {
         setQueryParam("sortDirection", sort.direction)
         setQueryParam("sortField", sort.field as string)
+        setQueryParam("hasCollaborator", sort.hasCollaborator)
```

Comment:
Do not touch sort

Reply:
@bkmaibach Removed the code from create-search